Tests/SwiftParserTest/translated/PoundAssertTests.swift

```diff
@@ -52,7 +49,7 @@ final class PoundAssertTests: XCTestCase {
       #assert(1️⃣, "error message")
       """#,
       diagnostics: [
-        DiagnosticSpec(message: "expected condition in '#assert' directive")
+        DiagnosticSpec(message: "expected value in pound literal declaration")
```

Comment:
Shouldn’t this be `expected value in macro expansion`? I think you need to update the `name_for_diagnostics` for `MacroExpansionDecl`.

Reply:
I suppose we can do that now. Previously, we decided to call them "pound literals" because we macros weren't a thing yet.